### PR TITLE
Change "Exit ATM MODE" from rectangle to button

### PR DESCRIPTION
**Pull request by tlindi**
Proposal for more streamlined visuals for ATM mode

**Current legend tpos-ATM mode**
image

**PR provided tpos ATM mode**
image

### CODE REVIEW

**Comment by tlindi** [comment]
Comments?

Could/should withdraw currence values be shown prefix "-" and color "negative"? But that would be another issue & PR.

**Review by talvasconcelos — APPROVED** [review]

File: templates/tpos/tpos.html [hunk 1]
```diff
@@ -39,6 +39,15 @@ <h5>
           </div>
         </div>
       </div>
+      <div class="col-12 col-sm-8 col-md-6 col-lg-4 text-center">
+        <q-btn
+          @click="() => {atmMode = false; getRates()}"
+          v-if="atmMode"
+          rounded
+          color="negative"
+          >ATM MODE</div
```
[the remark this answers is not in the document]
Comment (tlindi) [reply templates/tpos/tpos.html:48]: Would "EXIT ATM" be more clear?

This whole botton would not be need if value had negative color, and - sign on ATM mode and ATM button color presenting the mode IMHO.

**Comment by prusnak** [comment]
Which of these two red buttons exit the ATM mode?

Screenshot 2024-03-06 at 11 19 24

Only top, only right, both?

I suggest we use only one element and drop the other one. Maybe if the button on the right disappeared when the ATM mode is active, it would make it less confusing?

**Comment by tlindi** [comment]
Both. Disappearing button would be easy and works for me. Decision?

**Comment by tlindi** [comment]
> Which of these two red buttons exit the ATM mode?
> 
> Screenshot 2024-03-06 at 11 19 24
> Only top, only right, both?
> 
> I suggest we use only one element and drop the other one. Maybe if the button on the right disappeared when the ATM mode is active, it would make it less confusing?

did it. Look clean to me.

https://github.com/lnbits/tpos/assets/7249767/36bec4f3-7540-47ce-b158-9b50278ef609

**Comment by tlindi** [comment]
includes workaround for https://github.com/lnbits/tpos/issues/74

File: templates/tpos/tpos.html [hunk 2]
```diff
@@ -412,13 +432,6 @@ <h5 v-show="!denomIsSats" class="q-mt-none q-mb-sm">
         ></q-page-sticky
       >
     </template>
-    <q-btn
-      v-show="atmMode"
-      color="negative"
-      @click="() => {atmMode = false; getRates()}"
```
Comment (talvasconcelos) [inline templates/tpos/tpos.html:418]: This is important!! rates may have a premium that needs to be restored to normal when leaving ATM mode!

Comment (tlindi) [reply templates/tpos/tpos.html:418]: But isn't it taken care of on above line 44?

File: templates/tpos/tpos.html [hunk 3]
```diff
@@ -197,12 +206,23 @@ <h5>
...
         <q-btn
           @click="atm"
-          v-if="withdrawamtposs > 0"
+          v-if="withdrawamtposs > 0 && !atmMode"
           fab
           icon="atm"
           color="primary"
         ></q-btn>
       </q-page-sticky>
+<!--
```
Comment (dni) [inline templates/tpos/tpos.html:215]: what is this comment?

File: templates/tpos/tpos.html (hunk 2 above)
Comment (dni) [reply templates/tpos/tpos.html:418]: yes

File: templates/tpos/tpos.html (hunk 3 above)
Comment (tlindi) [reply templates/tpos/tpos.html:215]: Removal of 2nd button during this PR development. That part of code is removed completely on later commit (https://github.com/lnbits/tpos/pull/68/commits/1366a57f3dd32e25459a8cba8a2c1d5f83c44beb)

Comment (dni) [reply templates/tpos/tpos.html:215]: its not in this pr though, can you remove it here?

Comment (tlindi) [reply templates/tpos/tpos.html]: > its not in this pr though, can you remove it here?
>
My bad. Now better?

**Review by dni — APPROVED** [review]

**Review by prusnak — APPROVED** [review]

**Pull request merged**